logic/multisignature.js

```diff
@@ -23,14 +23,15 @@ __private.unconfirmedSignatures = {};
  * @param {Object} logger
  */
 // Constructor
-function Multisignature (schema, network, transaction, logger) {
+function Multisignature (schema, network, transaction, account, logger) {
```

Comment:
Reason of all changes in this file?

Reply:
`library.logic.account` is being used in code